### ISSUE TITLE
Constant reporting of under voltage even when disabled.

### ISSUE THREAD

**Issue by Glassboy** [OctoPrint/OctoPrint-PiSupport#11]
Neither unchecking "Enable under voltage and overheat detection via " or enabling "Disable warning about undervoltage on print start" stops alerts for  undervoltage.  The only method that works is removing alerts via config.txt.  The problem started occurring when the Pi support was split off into a separate project.  Before then only an occasional startup under voltage message would happen.

Also options have both "under voltage" (correct) and "undervoltage" (incorrect) used.

**Comment by kantlivelong** [comment]
Can you provide logs that showcase the issue? Also include `dmesg` log.

**Comment by Glassboy** [comment]
Just Octoprint.log?

**Comment by kantlivelong** [comment]
https://community.octoprint.org/t/what-is-a-systeminfo-bundle-and-how-can-i-obtain-one/29887

**Comment by Glassboy** [comment]
pi@bogle:~ $ /usr/bin/vcgencmd get_throttled
throttled=0x50000

The above is start after a reboot and I believe is a start up under voltage.  I don't see under voltages ongoing running vcgencmd from time to time.

**Comment by kantlivelong** [comment]
Right, the settings you are referring to only apply within OctoPrint. In other words they don't disable anything with under-voltage detection from a system standpoint.

**Comment by Glassboy** [comment]
[octoprint-systeminfo-20220329130918.zip](https://github.com/OctoPrint/OctoPrint-PiSupport/files/8367066/octoprint-systeminfo-20220329130918.zip)

**Comment by Glassboy** [comment]
> Right, the settings you are referring to only apply within OctoPrint. In other words they don't disable anything with under-voltage detection from a system standpoint.

It's only Octoprint that's annoying me, which is why I want to turn off the warning.  I've had my power supply checked by an electrician and I'm comfortable with the start up undervoltage.

**Comment by kantlivelong** [comment]
Thanks, which undervoltage related settings were you using when that log was created?

Also I'm not sure which tools the electrician used but electricians generally use a multi-meter which is not sufficient for picking up voltage dips that would affect the Pi.

**Comment by Glassboy** [comment]
![Screenshot from 2022-03-29 13-19-14](https://user-images.githubusercontent.com/894463/160508227-07d99e69-6a38-4331-bc3d-32e8ba125ac5.png)

**Comment by kantlivelong** [comment]
After unchecking "Enable under voltage and overheat detection via vcgencmd get_throttled" did you restart OctoPrint?

**Comment by Glassboy** [comment]
> After unchecking "Enable under voltage and overheat detection via vcgencmd get_throttled" did you restart OctoPrint?

The whole Pi has been restarted several times since that was unchecked and this last test was done.

**Comment by kantlivelong** [comment]
Thanks.

@foosel I'm not 100% sure but perhaps there's something sending an GET request to the plugin which in turn ends up calling `get_additional_environment`->`_check_throttled_state`->`send_plugin_message` to ultimately be displayed on the UI. If that's the case I think simply checking the setting and skipping the log entries or at least the `send_plugin_message` bit would do it.

**Comment by foosel** [comment]
@kantlivelong sounds like that might be the reason indeed...
